test/onnx/test_fx_to_onnx.py

```diff
@@ -3,6 +3,7 @@
 
 import pytorch_test_common
 import torch
+from parameterized import parameterized  # type: ignore[import]
```

Comment:
Possible to use pytorch's parameterization: https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_utils.py#L334 ?

Reply:
Then instantiate the test class with https://github.com/pytorch/pytorch/blob/709c9b5c93901be575b7abc71e9f8a5283fb76a2/torch/testing/_internal/common_utils.py#L251

Reply:
Should be doable.  Does it provide a better performace (speed up) on parametrized tests?

Reply:
No, but it is more conventional in this project.

Reply:
Done